eman2/protocols/tomo_protocols/protocol_tomo_boxing.py

```diff
@@ -0,0 +1,112 @@
+# **************************************************************************
+# *
+# * Authors:     Adrian Quintana (adrian@eyeseetea.com) [1]
+# *
+# * [1] EyeSeeTea Ltd, London, UK
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+
+from pyworkflow.utils.properties import Message
+from pyworkflow.gui.dialog import askYesNo
+from pyworkflow.protocol.params import BooleanParam, PointerParam, LEVEL_ADVANCED, EnumParam
+
+import eman2
```

Comment:
unused import

Reply:
Fixed in https://github.com/scipion-em/scipion-em-eman2/pull/64/commits/b8cce0b55532f87e72a098ef00c61cc64d975e2b